exercises/two-fer/TwoFer.vb

```diff
@@ -0,0 +1,7 @@
+﻿Public Module TwoFer
+    ' In order to get the tests running, first you need to make sure the Speak method 
```

Comment:
In general, we don't use comments in the stub to explain what the student should be doing. That is being done in the README.md and the tests file. However, from the config.json entries, it looks like this is the very first exercise that students will work on (after the hello-world one), so it might be good to have it here. I'd not do this for any of the subsequent exercises though, unless the comments are essential.